Aztec/Classes/TextKit/TextView.swift

```diff
@@ -1105,8 +1105,12 @@ open class TextView: UITextView {
         toggle(formatter: formatter, atRange: range)
 
         let liFormatter = LiFormatter(placeholderAttributes: typingAttributes)
-        toggle(formatter: liFormatter, atRange: range)
+        let isOlTagPresent = formatter.present(in: storage, at: range)
+        let isLiTagPresent = liFormatter.present(in: storage, at: range)
 
+        if (!isOlTagPresent && isLiTagPresent) || (isOlTagPresent && !isLiTagPresent) {
```

Comment:
Could this be refactored to an exclusive OR?

e.g. `if (isOlTagPresent != isLiTagPresent) {`

Reply:
Make the change

Reply:
Thanks! And good catch on dropping the parenthesis.